react_front_end/src/pages/components/TriangleLogo.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography";
+import logo from "../../assets/vir-us_triangle_logo_white.jpeg";
+import Grid from "@material-ui/core/Grid";

```

Comment:
Don't use the grid here as a solution to not using flex-box. We should pair up on this.